### PR TITLE
Remove the duration of the fixation in games that do not use the fixation #1368

### PR DESCRIPTION
**Pull request by Iluvrog**
I hope I don't forget a game

### CODE REVIEW

**Review by PillouMatou — CHANGES_REQUESTED** [review]
For the game Spot the differences, there is a fixation on the button at the bottom of the page.

The rest is fine 👍

**Review by PillouMatou — APPROVED** [review]
It's fine for me, @kuhlkrein @schwabdidier

File: gazeplay-core/src/main/java/net/gazeplay/ui/scenes/ingame/GameContext.java [hunk 1]
```diff
@@ -447,4 +452,8 @@ public Pane getRoot() {
     public void onGameStarted() {
     }
 
+    public void SetOffFixationLengthControl(){
```
Comment (kuhlkrein) [inline gazeplay-core/src/main/java/net/gazeplay/ui/scenes/ingame/GameContext.java:455]: Please don't use majuscule at the beginning of the function or PMD will detect is as an issue

File: gazeplay-games/src/main/java/net/gazeplay/games/moles/Moles.java [hunk 2]
```diff
@@ -145,6 +145,8 @@ public Color visitDark() {
         gameContext.getGazeDeviceManager().addStats(stats);
         play();
 
+        gameContext.SetOffFixationLengthControl();
+
```
Comment (kuhlkrein) [inline gazeplay-games/src/main/java/net/gazeplay/games/moles/Moles.java:149]: remove useless space

File: gazeplay-games/src/main/java/net/gazeplay/games/race/Race.java [hunk 3]
```diff
@@ -365,6 +365,8 @@ public void launch() {
         stats.notifyNewRoundReady();
         gameContext.getGazeDeviceManager().addStats(stats);
 
+        gameContext.SetOffFixationLengthControl();
+
```
Comment (kuhlkrein) [inline gazeplay-games/src/main/java/net/gazeplay/games/race/Race.java:369]: remove useless space

File: gazeplay-games/src/main/java/net/gazeplay/games/shooter/Shooter.java [hunk 4]
```diff
@@ -395,6 +395,8 @@ public void launch() {
         box.setTranslateX(0);
         box.setTranslateY(0);
 
+        gameContext.SetOffFixationLengthControl();
+
```
Comment (kuhlkrein) [inline gazeplay-games/src/main/java/net/gazeplay/games/shooter/Shooter.java:399]: remove useless space

**Review by kuhlkrein — CHANGES_REQUESTED** [review]

**Comment by Iluvrog** [comment]
@kuhlkrein it's done

**Comment by kuhlkrein** [comment]
please fix the conflicts and it will be fine :)

**Comment by Iluvrog** [comment]
> please fix the conflicts and it will be fine :)

done

**Review by kuhlkrein — APPROVED** [review]
Looks good to me !

**Pull request merged**